README.md

```diff
@@ -1 +1,55 @@
-# metrics-webapp
\ No newline at end of file
+# Covid-19 Metrics
+An app to know worldwide total confirm cases of Covid-19.
+
+## Built With
+
+- HTML, CSS, Javascript
+- Webpack, Stylelint, ESlint, React JS, Redux, Router, Jest, Github Actions
+
+## Live Demo
+
+[https://covid19-matrics-nayankaran.netlify.app/](https://covid19-matrics-nayankaran.netlify.app/)
+
```

Comment:
- [ ] Kindly add a link for a recorded video presenting the features of the project.

Reply:
Thank you for your suggestion. I added presentation video to the readme file.

Reply:
Addressed ✔